src/audio/kpb.c

```diff
@@ -817,9 +818,14 @@ static uint64_t kpb_draining_task(void *arg)
 	size_t size_to_read;
 	size_t size_to_copy;
 	bool move_buffer = false;
+	static uint32_t drained;
```

Comment:
Why static? Shouldn't it be reset for the consecutive runs?

Reply:
Yes I was testing different draining mechanism for the release which exit draining when there is no `sink->free`. That's why defined this variable static and forgot to change storage class.